src/player.ts

```diff
@@ -28,6 +28,81 @@ export interface PlayerConfig extends NativeInstanceConfig {
    * ```
    */
   licenseKey?: string;
+  /**
+   * Configuration options for player's default playback behavior.
```

Comment:
Please always use the existing documentation from the native SDKs to ensure consistency.

```suggestion
   * Configures playback behaviour. A default PlaybackConfig is set initially.
```